spanishdict/equivalency
# Feature 180463518 punctuation as whitespace rule

@vincecampanale 

# What/Why
Create a new rule that replaces punctuation tokens with a space.
https://www.pivotaltracker.com/story/show/180463518

# How to test
* Run unit tests locally `yarn test`
* Verify that new tests described by `Punctuation As Whitespace Rule` pass

_note: manual testing will be easier when the rule is included in lessons_
